### PR TITLE
Handle token info and apply formatting

### PR DESCRIPTION
**Pull request by neithanmo**
changes:

   - add add a list of token(white listed)
   - if token is present format it using the info in the list for amount and fees
   - if token is not present, indicates that value is raw
   - update tests

### CODE REVIEW

File: app/src/token_info.rs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            "decimals": 8,
+            "logoURI": "https://chainport-public-files-prod.s3.amazonaws.com/61bb7212-13ab-4a9a-a7c2-16a9583d2342.svg",
+            "website": "https://app.chainport.io"
+        },
+        {
+            "identifier": "5d43c6f2bbd70ae658661ab58f138b4d54276dd9102966b486bcc0d6e3ab89a3",
+            "symbol": "TINE",
+            "decimals": 8,
+            "logoURI": "https://chainport-public-files-prod.s3.amazonaws.com/61bb7212-13ab-4a9a-a7c2-16a9583d2342.svg",
+            "website": "https://app.chainport.io"
```
Comment (emmanuelm41) [inline app/src/token_info.rs]: These two are just test values as they said, so I would just leave the IRON value

File: app/src/token_info.rs [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+pub static TOKEN_LIST: &str = r#"
+{
+    "schemaVersion": 2,
+    "assets": [
+        {
+            "identifier": "51f33a2f14f92735e562dc658a5639279ddca3d5079a6d1242b2a588a9cbf44c",
+            "symbol": "IRON",
+            "decimals": 8,
+            "logoURI": "https://ironfish.network/favicon.ico",
+            "website": "https://ironfish.network"
```
Comment (emmanuelm41) [inline app/src/token_info.rs:14]: Do we really need these values? Are we going to use them somehow? If not, maybe we can save some space

File: app/src/utils/int_format.rs
Comment (emmanuelm41) [inline app/src/utils/int_format.rs:1]: This file seems like a really important one, as it actually formats the raw value. Is it possible to add some unit testing on rust with different cases? As many as possible.

File: app/src/parser/transaction.rs [hunk 3]
```diff
@@ -198,6 +186,62 @@ impl<'a> Transaction<'a> {
         Ok(fields)
     }
 
+    fn format_output(
```
Comment (emmanuelm41) [inline app/src/parser/transaction.rs:189]: Maybe some rust tests for this too?

**Review by emmanuelm41 — COMMENTED** [review]
Overall, it seems like an amazing progress!!

File: app/src/token_info.rs (hunk 1 above)
Comment (neithanmo) [reply app/src/token_info.rs]: done!

File: app/src/parser/transaction.rs (hunk 3 above)
Comment (neithanmo) [reply app/src/parser/transaction.rs:189]: the unit tests in the int_format module are sufficient as it is what that function uses

**Review by emmanuelm41 — APPROVED** [review]

**Pull request merged**